### DISCUSSION TITLE
%CMI_USER_AFKKICK_IN% placeholder

### DISCUSSION THREAD

**Discussion by MrMcyeet** [Zrips/CMI#4493] [Feature Requests]
**Description of feature/improvement request:**
This feature/placeholder would output the amount of time till they get kicked for afk.

---
### Detailed explanation of what it should do, when and in what situation: ###
Basically an addition for the cmi.command.afk.kickOutIn.[seconds] permission that you added, but this is a placeholder that would output the amount of time till player would be kicked out

### Basic explanation as to why this feature is needed and/or useful to have: ###
Useful for notifications or messages. also would be useful for server staff.

**Comment by ghost** [comment]
I mean, currently you can use a bit of default PlaceholderAPI placeholders such as formatter (etc) to achieve this, but the only issue is that you need to know how long the auto afk kick takes, correct?

If I'm wrong here, I would love to see this added then.
